app/controllers/conversations_controller.rb

```diff
@@ -30,12 +30,22 @@ def index
   end
 
   def create
-    contact_ids = params[:contact_ids]
-
-    # Can't split nil
-    if contact_ids
-      contact_ids = contact_ids.split(',') if contact_ids.is_a? String
-      person_ids = current_user.contacts.where(id: contact_ids).pluck(:person_id)
+    contacts_ids = params["contacts-ids"] || params["contacts-ids-mobile"]
+
+    # Contacts autocomplete does not work the same way on mobile and desktop
+    # Mobile returns contact ids array while desktop returns person id
+    # This will have to be removed when mobile autocomplete is ported to Typeahead
+    if defined? params["contacts-ids-mobile"]
+      # Can't split nil
+      if contacts_ids
+        contacts_ids = contacts_ids.split(',') if contacts_ids.is_a? String
+        person_ids = current_user.contacts.where(id: contacts_ids).pluck(:person_id)
+      end
+    else
+      if contacts_ids
+        contacts_ids = contacts_ids.split(',') if contacts_ids.is_a? String
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.